config/command_aliases.json

```diff
@@ -0,0 +1,3 @@
+{

```

Comment:
This file should probably follow the lead of the other config files and be renamed to something like `command_aliases_example.json` , with `command_aliases.json` being ignored in `.gitignore`. That way, we don't overwrite the command aliases configuration when there is a new bot update.

Reply:
Created `example_command_aliases.json` and added `config/command_aliases.json` to `.gitignore`